frontend/src/main/components/Courses/CoursesTable.js

```diff
@@ -67,6 +71,10 @@ import React from "react";
      ];
 
      if (hasRole(currentUser, "ROLE_ADMIN") || hasRole(currentUser, "ROLE_INSTRUCTOR")) {
+         if(showEnabled){
```

Comment:
I suggest adding a similar prop called `deleteEnabled` because we don't want a delete button on the show page.

If you aren't sure why: try clicking the delete button on the show page right now and see what happens (spoiler alert: it doesnt' appear to do anything, but the second time you click it if fails because the delete went through; but we are still looking at a show page for a course that no longer exists.   We could fix that, but why?   Deleting a course should be rare anyway, so no need to make it terribly convenient.)

Reply:
Sounds good!